src/fontra/client/web-components/ui-form.js

```diff
@@ -78,10 +82,30 @@ export class Form extends SimpleElement {
       white-space: normal;
     }
 
-    .ui-form-value.edit-number-x-y {
+    .ui-form-value.edit-number-x-y,
+    .ui-form-value.icons {
       display: flex;
       gap: 0.3rem;
     }
+
+    .ui-form-icon {
+      overflow-x: unset;
+      width: 1.2em;
+      white-space: nowrap;
+      margin-left: 1.3em;
+      margin-right: 1.3em;
+    }
+
+    .ui-form-icon.ui-form-icon-button {
+      display: inline-block;
+    }
+
+    .ui-form-center {
```

Comment:
Perhaps this could be integrated into the `origin-radio-buttons` CSS?

Reply:
It's newly introduced because of the radio button icon, but it's more general for centering an element. Do you still want to move it to the panel-transformation?

Reply:
Ok, I see, let's leave it here.